app/helpers/tags_helper.rb

```diff
@@ -218,6 +213,30 @@ def blurb_tag_block(item, tag_groups=nil)
     tag_block.html_safe
   end
 
+  # Takes a tag category class name, e.g. Relationship, and returns a string.
+  # The returned string is plural and used for things other than the HTML class
```

Comment:
I see the result of `tag_block_class_name` being used as a class in `blurb_tag_block`. Should this comment be changed?

Reply:
Sure! I meant it was being used as not just an HTML class, but didn't phrase it clearly.